app/javascript/mastodon/features/ui/components/link_footer.js

```diff
@@ -79,7 +79,7 @@ class LinkFooter extends React.PureComponent {
           {' '}
           <a href='https://joinmastodon.org' target='_blank'><FormattedMessage id='footer.about' defaultMessage='About' /></a>
           {' · '}
-          <a href='https://joinmastodon.org/apps' target='_blank'><FormattedMessage id='footer.get_app' defaultMessage='Get the app' /></a>
+          <a href='https://getmammoth.app' target='_blank'><FormattedMessage id='footer.get_app' defaultMessage='Get the app' /></a>
```

Comment:
Perhaps we should be a bit more specific and instead say "Get the Mammoth app", as this is on the same line as other links to joinmastodon. We could perhaps also rename 'About' to 'About Mastodon'.

Reply:
good thinking